compile.c

```diff
@@ -4348,10 +4346,33 @@ compile_array(rb_iseq_t *iseq, LINK_ANCHOR *const ret, const NODE *node, int pop
     return 1;
 }
 
+/* Compile an array containing the single element represented by node */
+static int
+compile_array_1(rb_iseq_t *iseq, LINK_ANCHOR *const ret, const NODE *node, int popped)
+{
+    if (popped) {
```

Comment:
Seems `popped` never true?

Reply:
It's not. I was trying to be consistent with `compile_array`, but probably should not add code we don't use.